utils.js

```diff
@@ -0,0 +1,76 @@
+function generateSummary(core, metrics) {
```

Comment:
Please, add documentation on the module level and document at least the exported functions.

Reply:
done